queries.sql

```diff
@@ -0,0 +1,104 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+/*Part One*/
+SELECT * FROM animals
+WHERE name LIKE '%mon';
+
+SELECT name FROM animals
+WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name FROM animals
+WHERE neutered = true AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals
+WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals
+WHERE weight_kg > 10.5;
+
+SELECT * FROM animals
+WHERE neutered = true;
+
+SELECT * FROM animals
+WHERE name != 'Gabumon';
+
+SELECT * FROM animals
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/*Part Two*/
+-- UPDATE SPECIES WITH ROLLBACK
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+-- UPDATE SPECIES WITHOUT ROLLBACK 
+BEGIN TRANSACTION;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
```

Comment:
- [ ]  @tsohleDev kindly add code to verify that the change was made before and after the commit.